server/api/expense-pool/index.js

```diff
@@ -18,6 +18,14 @@ router.get('/:id', isPoolMember(), controller.getPool);
  */
 router.post('/:id/users', isPoolMember(), controller.addUserToPool);
 
+// ------------ Routes above have updated controllers ----------------
+
+// GET => show all pools
+// Added for testing only
+router.get('/test', controller.getPools);
```

Comment:
This  route is also not called in this change so lets leave out the changes to this file.